### PR TITLE
[16.0] [OU-ADD] account_sale_timesheet: Nothing to do

### PR DESCRIPTION
**Pull request by NgoHoangHung**

### CODE REVIEW

File: docsource/modules150-160.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | account_qr_code_sepa                            |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| account_sale_timesheet                    |                      |                                                 |
+| |del| account_sale_timesheet                    |Done                  |Merged into sale_project                         |
```
Comment (hoangtiendung070797) [inline docsource/modules150-160.rst]:
```suggestion
| |del| account_sale_timesheet                    |                      |Merged into sale_project                         |
```
@NgoHoangHung  It should be left blank

Comment (NgoHoangHung) [reply docsource/modules150-160.rst]: ok, I edited again

Comment (daiduongnguyen-odoo) [reply docsource/modules150-160.rst]: No , it should be `Done` not blank, because you need to declare it at `merged` section in `apriory.py` file

**Pull request closed without merging**